includes/privacy.php

```diff
@@ -0,0 +1,256 @@
+<?php
+
+namespace WordCamp\Budgets\Privacy;
+
+use WP_Query;
+use WordCamp\Budgets\Reimbursement_Requests;
+
+defined( 'WPINC' ) || die();
+
+
+add_filter( 'wp_privacy_personal_data_exporters', __NAMESPACE__ . '\register_personal_data_exporters' );
+add_filter( 'wp_privacy_personal_data_erasers', __NAMESPACE__ . '\register_personal_data_erasers' );
+
+/**
+ * Registers the personal data eraser for each WordCamp post type
+ * @param array $erasers
+ *
+ * @return array
+ */
+function register_personal_data_erasers( $erasers ) {
+	/**
+	 * We should not add a eraser for WordCamp post type, because it contains data which can be used for
+	 * accounting or reference purpose.
+	 *
+	 * This is just an empty stub so that we do not create one.
+	 */
+	return $erasers;
+}
+
+/**
+ * Registers the personal data exporter for each WordCamp post type.
+ *
+ * @param array $exporters
+ *
+ * @return array
+ */
+function register_personal_data_exporters( $exporters ) {
+	$exporters['wcb-reimbursements'] = array(
+		'exporter_friendly_name' => __( 'WordCamp Reimbursement Requests', 'wordcamporg' ),
+		'callback'               => __NAMESPACE__ . '\reimbursements_exporter',
+	);
+
+	$exporters['wcb-vendor-payments'] = array(
+		'exporter_friendly_name' => __( 'WordCamp Vendor Payment Requests', 'wordcamporg' ),
+		'callback' => __NAMESPACE__ . '\vendor_payment_exporter',
+	);
+
+	return $exporters;
+}
+
+/**
+ * Finds and exports personal data associated with an email address in a vendor payment request
+ *
+ * @param string $email_address
+ * @param int $page
+ *
+ * @return array
+ */
+function vendor_payment_exporter( $email_address, $page) {
+
+	$results = array(
+		'data' => array(),
+		'done' => true,
+	);
+
+	$user = get_user_by( 'email', $email_address );
+
+	if ( empty( $user->ID ) ) {
+		return $results;
+	}
+
+	$sponsor_invoices = wcb_get_post_wp_query( \WCP_Payment_Request::POST_TYPE, $page, $user->ID );
```

Comment:
Having some of the variable names reference sponsor invoices is kind of confusing, since that is a separate post type from vendor payments. They should probably reference `payments` instead.